packages/luvi/fix-re-not-found.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 7aa6eaa..0ce5058 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -157,9 +157,6 @@ if (WITH_LUA_ENGINE STREQUAL Lua)
+   add_definitions(-DWITH_PLAIN_LUA)
+ endif ()
+ 
+-if(LPEGLIB_DIR)
+-  set(lpeg_re_lua ${LPEGLIB_DIR}/re.lua)
+-endif()
+ lua_add_executable(luvi
+   ${winsvc}
+   src/main.c
+@@ -167,7 +164,7 @@ lua_add_executable(luvi
+   src/lua/init.lua
+   src/lua/luvipath.lua
+   src/lua/luvibundle.lua
+-  ${lpeg_re_lua}
++  deps/lpeg/re.lua
```

Comment:
We have a shared version of lua-lpeg available as either `lua51-lpeg` or `lua53-lpeg`.
Are you sure we can't replace this patch with `-DLPEGLIB_DIR=$TERMUX_PREFIX/share/lua/5.1/` in the `TERMUX_PKG_EXTRA_CONFIGURE_ARGS`?

Reply:
It won't work, the `lua_add_executable` macro will still concatenate even though it's absolute path. E.g `${TERMUX_PREFIX}/share/lua/5.1`, it would reproduce `${TERMUX_PKG_SRCDIR}/${TERMUX_PREFIX}/share/lua/5.1`. But shared `lpeg.so` is possible, although needs more patching on `deps/lpeg.cmake`.

Reply:
Hmm I see.